src/DataMapping.php

```diff
@@ -11,7 +11,7 @@
 class DataMapping
```

Comment:
This file should have been deleted (since yes, the new version is called `Mapping.php`)